0053-bundle-format/README.md

```diff
@@ -0,0 +1,483 @@
+
+### User Stories
+
+#### Story 1: Offline Sign - Offline Verify
+
+As an operator deploying Zarf in an airgapped network, I need to verify package signatures without any external network connectivity while maintaining full cryptographic verification of package authenticity. This maintains backwards compatibility with existing keypair signing.
+
+**Solution**: Use airgap profile (default) with keypair signing.
+
+```bash
+# Developer (online environment) - sign package
+zarf package sign zarf-package-app-amd64.tar.zst --signing-key cosign.key
+
+# Operator (airgap environment) - verify
+zarf package verify zarf-package-app-amd64.tar.zst --key cosign.pub
+```
+
+#### Story 2: Online Sign - Offline Verify
+
+As a package author using Zarf in a connected environment, I want to leverage the Sigstore public good infrastructure to provide additional assurance that package signatures are publicly recorded and auditable.
+
+**Solution**: Use online profile with embedded Trusted Root verification.
+
+```bash
+# Developer - sign with transparency log upload
+zarf package sign zarf-package-app-amd64.tar.zst --signing-key cosign.key --profile online
+
+# Operator - verify with embedded trusted root
+zarf package verify zarf-package-app-amd64.tar.zst --key cosign.pub
+```
+
+```bash
+# Developer - sign with keyless
+zarf package sign zarf-package-app-amd64.tar.zst --oidc-issuer='https://oauth2.sigstore.dev/auth' --profile online
+
+# Operator - verify with embedded trusted root - no external connectivity made
+zarf package verify zarf-package-app-amd64.tar.zst
+
+# Additionally verify with identity verification
+zarf package verify zarf-package-app-amd64.tar.zst \
+  --certificate-identity user@example.com \
+  --certificate-oidc-issuer https://oauth2.sigstore.dev/auth
+```
+
+#### Story 3: Online Sign - Online Verify - Private Sigstore Infrastructure
+
+As an enterprise running a private Sigstore instance for internal packages, I want to use a custom trusted root and private Fulcio/Rekor instances for signing and verification.
+
+**Solution**: Use online profile for signing and online verify with internal URLs or custom trusted root
+
+```bash
+
+# Sign with private infrastructure
+zarf package sign zarf-package-app-amd64.tar.zst \
+  --oidc-issuer https://oauth2.internal.company.com/auth \
+  --rekor-url https://rekor.internal.company.com \
+  --profile online
+
+# Verify with private infrastructure in an "online" profile
+Zarf package verify zarf-package-app-amd64.tar.zst \
+  --rekor-url https://rekor.internal.company.com \
+  --fulcio-url https://fulcio.internal.company.com \
+  --profile online
+
+# or verify in the default offline stance with a custom trusted-root
+# Create custom trusted root
+cosign trusted-root create \
+  --rekor-url https://rekor.internal.company.com \
+  --fulcio-url https://fulcio.internal.company.com \
+  --output custom_trusted_root.json
```

Comment:
Possibly outside of the scope of the proposal, but is there a point at which this command might become a zarf command?

Reply:
I mentioned a helper command for custom trusted roots in the `Version Skew`. I did not expand on all opportunities present - from exposing Cosign as an embedded command to other helper functions. Open to adding _some_ opportunities if we deem valuable.